recipes/eva-sub-cli/build.sh

```diff
@@ -3,11 +3,10 @@
 BIOVALIDATOR_VERSION=2.2.1
 
 EVA_SUB_CLI="${PREFIX}/share/${PKG_NAME}-${PKG_VERSION}"
-mkdir -p ${PREFIX}/bin ${EVA_SUB_CLI}
+mkdir -p ${EVA_SUB_CLI}
 
 # Install eva-sub-cli
 $PYTHON -m pip install .
-cp bin/* ${PREFIX}/bin
```

Comment:
How does it install the scripts in $PREFIX/bin now ?

Reply:
We've updated our build to fix some issues and also use a [toml file](https://github.com/EBIvariation/eva-sub-cli/blob/main/pyproject.toml#L32), so the `pip install` places the scripts appropriately now without needing to copy the bin folder. I've tested the recipe locally to confirm that the scripts can be found.